test/fixtures/test-runner/output/test-timeout-flag.js

```diff
@@ -0,0 +1,25 @@
+// Flags: --test-timeout=20
+'use strict';
+const { describe, test, afterEach } = require('node:test');
+const { setTimeout } = require('node:timers/promises');
+
+let timeout;
+
+describe('--test-timeout is set to 20ms', () => {
+  test('should timeout after 20ms', async () => {
+    timeout = await setTimeout(2000);
+  });
+  test('should timeout after 5ms', { timeout: 5 }, async () => {
+    timeout = await setTimeout(2000);
+  });
+  test('should not timeout', { timeout: 5000 }, async () => {
```

Comment:
Same comment here for the 5000ms `timeout` option. There is no harm making this really large since the timeout should never be hit. It also reduces the chance of any timer weirdness in the CI.